mitodl/mit-open
# New default image for learning resources

### Description (What does it do?)


Replaces the default image for learning resources with a new one.

Displays it in the learning resource drawer where no image is set on the resource.

### Screenshots (if appropriate):

<img width="670" alt="image" src="https://github.com/mitodl/mit-open/assets/939376/c84e20ac-6456-413b-912b-725ce70f7b7b">


### How can this be tested?


Find a resource that has no image and check that the default is displayed in its card, list and drawer components.

This one currently has no image:

/search/?q=No+Code+AI+and+Machine+Learning%3A+Building+Data+Science+Solutions
/search/?resource=14465
